community/kernel/src/main/java/org/neo4j/graphdb/schema/ConstraintCreator.java

```diff
@@ -47,6 +47,14 @@
     ConstraintCreator assertPropertyIsUnique( String propertyKey );
 
     /**
+     * Imposes an existence constraint for the given property, such that any node with the given label must have a
+     * value set for the given property key.
+     *
+     * @return a {@link ConstraintCreator} instance to be used for further interaction.
+     */
+    ConstraintCreator assertPropertyExists( String propertyKey );
```

Comment:
@akollegger What do you think of the name of this method?

In cypher it is `NOT NULL` so it might make sense to call this method `assertPropertyIsNotNull()`, but on the other hand the embedded java API does not behave the same as cypher, getting a property that is not set results in `null` in Cypher, but in the embedded java API it results in an exception.